sdk/eventhub/azure-eventhub/azure/eventhub/_consumer.py

```diff
@@ -198,10 +198,12 @@ def _open(self) -> bool:
             )
             self._handler = cast("ReceiveClient", self._handler)
             self._handler.open(connection=conn)
-            while not self._handler.client_ready():
-                time.sleep(0.05)
-            self.handler_ready = True
+            self.handler_ready = False
             self.running = True
+        
+        if not self.handler_ready:
```

Comment:
is `self.handler_ready` being used/set anywhere outside this? Since we are doing `self.handler_ready = False` a few lines above we might not need this check and only have the second if statement

Reply:
nope not used anywhere else!